src/model/ScheduleDay.hpp

```diff
@@ -66,8 +66,19 @@ namespace model {
     /// Returns a vector of values in the same order and with the same number of elements as times.
     std::vector<double> values() const;
 
-    /// Returns the value in effect at the given time.  If time is less than 0 days or greater than 1 day, 0 is returned.
-    double getValue(const openstudio::Time& time) const;
+    /// Returns the value in effect at the given time.
+    /// If time is less than 0 days or greater than 1 day, 0 is returned.
+    double getValue(const openstudio::Time& time, int numberOfTimestepsPerHour = 0) const;
```

Comment:
I find the numberOfTimestepPerHour argument weird.

The ScheduleDay is part of the model, so we **know** the Number of Timestep per Hour (`model.getTimestep.numberOfTimestepsPerHour`)